upup/pkg/fi/cloudup/awstasks/launchtemplate_target_api.go

```diff
@@ -55,6 +55,10 @@ func (t *LaunchTemplate) RenderAWS(c *awsup.AWSAPITarget, a, e, changes *LaunchT
 				Ipv6AddressCount:         t.IPv6AddressCount,
 			},
 		},
+		PrivateDnsNameOptions: &ec2.LaunchTemplatePrivateDnsNameOptionsRequest{
+			EnableResourceNameDnsAAAARecord: fi.Bool(*t.IPv6AddressCount > 0),
```

Comment:
Should we not enable A records too?

Reply:
They aren't default?

Reply:
They are for ipv6 native, but not sure about dual stack